src/management/projects.ts

```diff
@@ -423,3 +442,21 @@ export async function getFirebaseProject(projectId: string): Promise<FirebasePro
     );
   }
 }
+
+export interface ProjectInfo {
+  projectNumber: string;
```

Comment:
You know what would probably be _really_ nice, is making this, you know, a number. Similarly with the createTime - at _minimum_ that should be a number, but better yet might be that it's a Date.

Reply:
https://cloud.google.com/resource-manager/reference/rest/v1/projects#resource:-project Agree fully, but not up to me - the API returns these both as strings, so I'm matching that representation.